owncloudApp/src/main/java/com/owncloud/android/shares/ui/fragment/PublicShareDialogFragment.kt

```diff
@@ -61,75 +56,89 @@ import java.text.ParseException
 
     /**
      * File to share, received as a parameter in construction time
      */
-    private var mFile: OCFile? = null
+    private var file: OCFile? = null
 
     /**
-     * Existing share to update. If NULL, the dialog will create a new share for mFile.
+     * Existing share to update. If NULL, the dialog will create a new share for file.
      */
-    private var mPublicShare: RemoteShare? = null
+    private var publicShare: RemoteShare? = null
 
     /*
      * OC account holding the file to share, received as a parameter in construction time
      */
-    private var mAccount: Account? = null
+    private lateinit var account: Account
 
     /**
      * Reference to parent listener
      */
-    private var mListener: ShareFragmentListener? = null
+    private var listener: ShareFragmentListener? = null
 
     /**
      * Capabilities of the server
      */
-    private var mCapabilities: OCCapability? = null
+    private var capabilities: OCCapability? = null
 
     /**
      * Listener for changes in password switch
      */
-    private var mOnPasswordInteractionListener: OnPasswordInteractionListener? = null
+    private var onPasswordInteractionListener: OnPasswordInteractionListener? = null
 
     /**
      * Listener for changes in expiration date switch
      */
-    private var mOnExpirationDateInteractionListener: OnExpirationDateInteractionListener? = null
+    private var onExpirationDateInteractionListener: OnExpirationDateInteractionListener? = null
 
     /**
      * UI elements
      */
 
-    private var mNameSelectionLayout: LinearLayout? = null
-    private var mNameValueEdit: EditText? = null
-    private var mPasswordLabel: TextView? = null
-    private var mPasswordSwitch: SwitchCompat? = null
-    private var mPasswordValueEdit: EditText? = null
-    private var mExpirationDateLabel: TextView? = null
-    private var mExpirationDateSwitch: SwitchCompat? = null
-    private var mExpirationDateExplanationLabel: TextView? = null
-    private var mExpirationDateValueLabel: TextView? = null
-    private var mErrorMessageLabel: TextView? = null
-    private var mPermissionRadioGroup: RadioGroup? = null
-    private var mReadOnlyButton: RadioButton? = null
-    private var mReadWriteButton: RadioButton? = null
-    private var mUploadOnlyButton: RadioButton? = null
+    private var nameSelectionLayout: LinearLayout? = null
+    private var nameValueEdit: EditText? = null
+    private var passwordLabel: TextView? = null
+    private var passwordSwitch: SwitchCompat? = null
+    private var passwordValueEdit: EditText? = null
+    private var expirationDateLabel: TextView? = null
+    private var expirationDateSwitch: SwitchCompat? = null
+    private var expirationDateExplanationLabel: TextView? = null
+    private var expirationDateValueLabel: TextView? = null
+    private var errorMessageLabel: TextView? = null
+    private var permissionRadioGroup: RadioGroup? = null
```

Comment:
why don't you use synthetic? you can avoid declare and find all these views...

Reply:
Done